src/System.CommandLine.Tests/TokenTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using FluentAssertions;
+using System.CommandLine.Parsing;
+using Xunit;
+
+namespace System.CommandLine.Tests
```

Comment:
Are we leveraging file scoped namespaces?
I see at least one file was converted to use them by @jonsequitur in #2226